### ISSUE TITLE
NTR disruption of host extracellular matrix

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#25393]
For GO-PathGO alignment

References from @genegodbold 

Dentilisin is an acylated chymotrypsin-like protease complex (CTLP) found on the surface of the bacterium. Dentilisin can act on host pro-matrix metalloproteinase-2 (MMP-2), a Zn2+-dependent endopeptidase, secreted by host periodontal ligament cells to persistently activate it, thereby allowing host fibronectin, a component of the extracellular matrix, to be cleaved locally by the active MMP-2 [PMID:21115719]. Treponema denticola upregulates activation of MMP-2 in periodontal ligament cells [PMID:24973519]. Dentilisin can degrade gelatin, laminin, type IV collagen, and serum components [PMID:2404867][PMID:7543886].

**Comment by pgaudet** [comment]
+[Term]
+id: GO:0141041
+name: disruption of extracellular matrix of another organism
+namespace: biological_process
+namespace: The chemical reactions and pathways performed by an organism resulting in the breakdown of the extracellular matrix of another organism. {xref="PMID:21115719", xref="PMID:2817904", xref="PMID:2404867", xref="PMID:25865874"}
+comment: Note that this process can occur during host-symbiont interactions or be mediated by toxin secreted by free-living organisms.
+is_a: GO:0044419 ! biological process involved in interspecies interaction between organisms
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/25393" xsd:anyURI
+created_by: pg
+creation_date: 2023-05-17T13:43:30Z

**Comment by genegodbold** [comment]
Hey @pgaudet, I don't know if you need another one but here is one:

Alp1 from the fungal pathogen Neosartorya fumigata (formerly Aspergillus fumigatus) is a serine protease that promotes airway hyper-responsiveness, inducing bronchoconstriction in lung slices from mice. It enhances calcium mobilization in airway smooth muscle cells. Alp1 **degrades host integrin-mediated extracellular matrix** via its protease activity [PMID:25865874].

**Comment by pgaudet** [comment]
Obsoleted as unnecessary grouping term (2 annotations) 
Consider [venom-mediated disruption of extracellular matrix in another organism](http://purl.obolibrary.org/obo/GO_0044523)
and [symbiont-mediated disruption of host extracellular matrix](http://purl.obolibrary.org/obo/GO_0141066)

**Comment by genegodbold** [comment]
@pgaudet I agree that those two proposed terms should be instituted (venom-mediated AND symbiont-mediated disruption of host ECM)
